<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Smoke Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="40.311">
  <properties />
            <testcase name="buildYourOwnComputer" time="18.952" classname="com.nopcommerce.demo.testsuite.TestSuite">
                <failure type="java.lang.AssertionError"
                  message="expected [Build your ow computer] but found [Build your own computer]"
              >
<![CDATA[java.lang.AssertionError: expected [Build your ow computer] but found [Build your own computer]
  org.testng.Assert.fail(Assert.java:97)
  org.testng.Assert.assertEqualsImpl(Assert.java:136)
  org.testng.Assert.assertEquals(Assert.java:118)
  org.testng.Assert.assertEquals(Assert.java:575)
  org.testng.Assert.assertEquals(Assert.java:585)
  com.nopcommerce.demo.testsuite.TestSuite.buildYourOwnComputer(TestSuite.java:62)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:766)
  org.testng.TestRunner.run(TestRunner.java:587)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
  org.testng.TestNG.runSuites(TestNG.java:1039)
  org.testng.TestNG.run(TestNG.java:1007)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="clickOnDesktopAndSortByAndAddToCart" time="15.207" classname="com.nopcommerce.demo.testsuite.TestSuite" />
                <testcase name="clickOnDesktopAndSortByZtoA" time="6.152" classname="com.nopcommerce.demo.testsuite.TestSuite" />
      
</testsuite>
